src/components/Navbar/index.jsx

```diff
@@ -33,27 +37,64 @@ const socialIcons = [
 ]
 
 export default function Navbar() {
+  const { toggleTheme, theme } = useTheme()
+  const isChecked = theme === "dark"
+  const { isAuthenticated } = useAuth()
+  const pathname = usePathname()
+
+  const [isOpen, setOpen] = useState(false)
+  const handleOpen = () => {
+    setOpen(!isOpen)
+  }
+
+  useEffect(() => {
+    setOpen(false)
+  }, [pathname])
+
   return (
-    <S.Main>
-      {/* Div que contém os ícones */}
-      <S.Icons>
-        {socialIcons.map((socialIcon, index) => (
-          <S.LinkIcons href={socialIcon.link} key={index}>
-            <Image src={socialIcon.image} alt={socialIcon.name} fill />
-          </S.LinkIcons>
-        ))}
-      </S.Icons>
-
-      {/* Div que contém o título */}
-      <S.Title>blog-ui</S.Title>
-
-      {/* Div que contém os botões */}
-      <S.Buttons>
-        <S.LinkPages href="/">Home</S.LinkPages>
-        <S.LinkPages href="/contact">Contact</S.LinkPages>
-        <S.LinkPages href="/about">About</S.LinkPages>
-        <S.LinkPages href="/login">Login</S.LinkPages>
-      </S.Buttons>
-    </S.Main>
+    <>
+      <S.Main>
+        <S.Icons>
+          {socialIcons.map((socialIcon, index) => (
+            <S.LinkIcons href={socialIcon.link} key={index}>
+              <Image src={socialIcon.image} alt={socialIcon.name} fill />
+            </S.LinkIcons>
+          ))}
+        </S.Icons>
+
+        <S.Title>blog-ui</S.Title>
+
+        <S.Buttons>
+          <Switch {...""} onChange={() => toggleTheme()} checked={isChecked} />
```

Comment:
what is this `{...""}`? This is not doing anything apparently. Also, since you are not passing anything to `toggleTheme` you might want to just call it `onChange={toggleTheme}`.